CONTRIBUTING.md

```diff
@@ -33,6 +33,7 @@ This `egghead-ui` repo does two things:
 - `yarn dev:library` to develop the library
 - `yarn verify` check linting and builds work
 - [localhost:2000](http://localhost:2000) to view the style guide
+- run `yarn build` to build updated components
```

Comment:
Yeah, I just was running `dev:styleguide` instead to visually see it and left it running...  Took this out.